pcli/src/main.rs

```diff
@@ -58,7 +58,7 @@ impl App {
             .transpose()?
             .ok_or_else(|| anyhow::anyhow!("view service did not report sync status"))?;
 
-        println!(
+        eprintln!(
```

Comment:
All our other output goes to stdout, do we want this one specifically to go to stderr?

Reply:
Fair point, not strictly necessary here. I'll remove that change from this PR, and open a separate issue to discuss the stderr/stdout inconsistency.

Reply:
No worries, I wasn't concerned about it not being necessary here, just wanting to make sure if we do that change, we do it completely and not just here.